zerver/migrations/0644_merge_20241219_1316.py

```diff
@@ -0,0 +1,13 @@
+# Generated by Django 5.0.9 on 2024-12-19 13:16
+
+from django.db import migrations
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ("zerver", "0624_rename_recent_topics"),
+        ("zerver", "0643_realm_scheduled_deletion_date"),
+    ]
+
+    operations = []
```

Comment:
This files gets generated after I rebase my local branch against the latest upstream.